packages/interwovenkit-react/src/pages/deposit/address/DepositAddress.module.css

```
@@ -0,0 +1,191 @@
+
+.selectors {
+  display: flex;
+  gap: 16px;
+
+  width: 100%;
+}
+
+.selectorGroup {
+  flex: 1;
+
+  display: flex;
```

Stylelint (17.14.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)